sirepo/template/srw.py

```diff
@@ -1967,18 +1941,25 @@ def _rotate_report(report, ar2d, x_range, y_range, info):
     return ar2d, x_range, y_range
 
 
-def _rsopt_jinja_context(model):
+def _rsopt_jinja_context(data):
     import multiprocessing
+    model = data.models[_RSOPT_EXPORT_BASE_NAME]
     e = _process_rsopt_elements(model.elements)
     return PKDict(
+        fileBase=_RSOPT_EXPORT_BASE_NAME,
         forRSOpt=True,
+        libFiles=_SIM_DATA.lib_file_basenames(data),
         numCores=int(model.numCores),
         numWorkers=max(1, multiprocessing.cpu_count() - 1),
         numSamples=int(model.numSamples),
+        outFileName=f'{_RSOPT_EXPORT_BASE_NAME}.out',
+        randomSeed=model.randomSeed if model.randomSeed is not None else '',
+        readmeFileName='README.txt',
         rsOptElements=e,
         rsOptParams=_RSOPT_PARAMS,
         rsOptParamsNoRot=_RSOPT_PARAMS_NO_ROTATION,
         rsOptOutFileName='scan_results',
+        runDir=f'{_RSOPT_EXPORT_BASE_NAME}_scan',
```

Comment:
Why isn't this just `scan`? Why does _RSOPT_EXPORT_BASE_NAME need to be part of the name? Isn't it in the rsopt directory?